README.md

```diff
@@ -1,5 +1,17 @@
 # trace-server-protocol
 
+The Trace Server Protocol is a protocol designed to allow frontend visualizers to be decoupled from backend. It allows the frontend to query resources in a RESTful way. The guiding principles:
+
+* Keep things as imutable as possible
+* A trace can be queried for any UI elements.
+ * UI elements can be but should not be limited to charts, graphs, trees and more.
```

Comment:
Markdownlint has issues with this reformatted line and the few ones below (8 and 9).